eZ/Publish/Core/Repository/Repository.php

```diff
@@ -802,7 +802,6 @@ public function getSearchService()
         $this->searchService = new SearchService(
             $this,
             $this->searchHandler->contentSearchHandler(),
```

Comment:
Yes. I kept the main search handler anticipating we'll eventually have other types of search, in that case it would aggregate more than just Content search handler.